2022/01/i-am-closer/input.txt

```diff
@@ -0,0 +1,2250 @@
+5800
```

Comment:
Can you remove this file? It should be different for others

Reply:
OK, I'll handle it later, along with my answer of Q2.